lib/parachain/statement_distribution_message_test.go

```diff
@@ -0,0 +1,199 @@
+package parachain
+
+import (
+	_ "embed"
+	"fmt"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
+)
+
+//go:embed testdata/statement_distribution_message.yaml
+var expectedSDMHexRaw string
+
+var expectedSDMHex map[string]string
+
+func init() {
+	err := yaml.Unmarshal([]byte(expectedSDMHexRaw), &expectedSDMHex)
+	if err != nil {
+		fmt.Printf("Error unmarshaling test data: %s\n", err)
+		return
+	}
+}
+
+func TestStatementDistributionMessage(t *testing.T) {
+	t.Parallel()
+
+	var collatorSignature CollatorSignature
+	tempSignature := common.MustHexToBytes("0xc67cb93bf0a36fcee3d29de8a6a69a759659680acf486475e0a2552a5fbed87e45adce5f290698d8596095722b33599227f7461f51af8617c8be74b894cf1b86") //nolint:lll
```

Comment:
Is there a reason for not moving this to yaml or you just missed it?

Reply:
Moving this to YAML wouldn't be beneficial as it's not so long string.

Reply:
hmm, I see that it is using `//nolint:lll` to skip the linter and I believe the limit is set to 125 characters. The signature has a length of 130. I feel we should move this as well